hca/upload/__init__.py

```diff
@@ -0,0 +1,51 @@
+import os
+import re
+
+from .config_store import ConfigStore
+from .upload_area import UploadArea
+from .upload_area_urn import UploadAreaURN
+from .exceptions import UploadException
+from .s3_agent import S3Agent
+
+UPLOAD_BUCKET_TEMPLATE = "org-humancellatlas-upload-%s"
```

Comment:
Can this be put in the ConfigStore, or better yet, retrieved from the upload service?

Also, can you name the parameter? (`"org-humancellatlas-upload-{stage}".format(stage="foo")`)